### PR TITLE
Abandoned flag for packages and versions

### PR DESCRIPTION
**Pull request by psampaz**
Packagist.org api includes abandoned flag both in versions (https://github.com/composer/packagist/blob/master/src/Packagist/WebBundle/Entity/Version.php)
and packages (https://github.com/composer/packagist/blob/master/src/Packagist/WebBundle/Entity/Package.php). 

You can see an example here: https://packagist.org/packages/nella/framework.json

If the package is not abandoned, the abandoned flag is not included in packagist.org api response.

This pull request adds support for this abandoned flag.

### CODE REVIEW

File: src/Packagist/Api/Result/Package.php [hunk 1]
```diff
@@ -50,6 +50,11 @@ class Package extends AbstractResult
     protected $favers;
 
     /**
+     * @var boolean
```
Comment (umpirsky) [inline src/Packagist/Api/Result/Package.php]: `bool`, same on other places. You can run PHP-CS-Fixer. See `phpdoc_scalar` on https://github.com/FriendsOfPHP/PHP-CS-Fixer.

**Comment by umpirsky** [comment]
@psampaz Cool addition, thanks @psampaz! :+1:

**Comment by psampaz** [comment]
@umpirsky fixed. Everything now is bool.

**Comment by umpirsky** [comment]
Merged, thanks @psampaz! :+1:

**Comment by psampaz** [comment]
@umpirsky thanks!

**Pull request merged**